### DISCUSSION TITLE
how to run gpus commandline

### DISCUSSION THREAD

**Discussion by manepallirajesh** [batect/batect#1364] [General]
Does batect support nvidia?
how to pass gpus flag during docker launch?

**Comment by charleskorn** [comment]
Hi @manepallirajesh, this isn't something that Batect currently supports. Could you share some more information about what you're trying to do?

**Reply by manepallirajesh** [reply]
Hi, @charleskorn, currently I am using docker-compose to support gpus inside the containers, using  https://docs.docker.com/compose/gpu-support/. However I could not find something similar in batect to pass nvidia runtime.
Requirement: is to get gpus access inside the running container. 
docker command line supports as part of launch option `--gpus`.
docker-compose supports runtime option

Need something similar in batect to get gpus access inside the container.
